tests/PayfastSDK/Request/RequestHandlerTest.php

```diff
@@ -96,10 +98,39 @@ public function testHandleRequest($options, $result)
             ),
         ]);
 
-        $request = $this->handler->handleRequest($options);
+        $response = $this->handler->custom('POST', '/adhoc', $options);
```

Comment:
What do you mean by 'custom'?

Reply:
custom request, instead of one of the baked in functions that will come later. Tips on a better name?

Reply:
Maybe just `request`?